examples/tutorials/Differentiation_Infrastructure_in_Deepchem.ipynb

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
<p>The font here is weird; can you make it the usual font for the tutorials?</p><p> </p><p>Make the title "Broyden's First Method" and not "Broyden1 method." Make sure capitalization etc is correct on titles for sections</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
i was using html tags, i will revert back to normal.